project_profile_counter.py

```diff
@@ -0,0 +1,28 @@
+import json
+import requests
+import collections
+
+SERVER = 'https://tokyo.sdelements.com'
+API_TOKEN = 'f258c47557a3f98f55d4fbd0cb9d8354c86fbb52'
+PROJECTS_URL = '{server}/api/v2/projects/?page_size=100'.format(server=SERVER)
+REQUESTS_HEADER = {
+    'Authorization': "Token " + API_TOKEN,   # Authorization header + generated API TOKEN
+    'Accept': "application/json"             # Accepts API response in JSON format
+}
+
+url = PROJECTS_URL
+profiles = []
+while True:
+    response = requests.get(url, headers=REQUESTS_HEADER) 
+    data = response.json()
+
+    for item in data['results']:    # loop through the values in the results key 
+        profiles.append(item['profile']['name'])    # add each profile's name to list
```

Comment:
You can do the looping as a list comprehension. Not sure what's nicer in this case.

```
profiles += [project['profile']['name'] for project in data['results']]
```

Either way you should use variable names that are descriptive. data['results'] is a list of projects, so you are looping over projects, and should call your variable project rather than item.

Reply:
Is there any advantage to using list comprehensions other than to condense code?

Reply:
List comprehensions are more "pythonic". There are also sometimes memory or performance wins when doing a list comprehension, but I'm not sure if that'd be the case here.